<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Weapon Detector</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_detection">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>550</width>
      <height>340</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>854</width>
      <height>480</height>
     </size>
    </property>
    <property name="text">
     <string>No Camera Displayed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stop_detection_button">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>760</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop Monitoring</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_detection2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>550</width>
      <height>340</height>
     </rect>
    </property>
    <property name="text">
     <string>No Camera Displayed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_detection3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>550</width>
      <height>340</height>
     </rect>
    </property>
    <property name="text">
     <string>No Camera Displayed</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_detection4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>400</y>
      <width>550</width>
      <height>340</height>
     </rect>
    </property>
    <property name="text">
     <string>No Camera Displayed</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
